src/cairoUtilFuncGen/storage/storageDelete.ts

```diff
@@ -126,12 +127,13 @@ export class StorageDeleteGen extends StringIndexedFuncGen {
     const cairoType = CairoType.fromSol(type, this.ast);
     return {
       name: funcName,
-      code: [
-        `func ${funcName}${IMPLICITS}(loc: felt){`,
-        ...mapRange(cairoType.width, (n) => `    WARP_STORAGE.write(${add('loc', n)}, 0);`),
-        `    return ();`,
-        `}`,
-      ].join('\n'),
+      code: endent`
+        fn ${funcName}(loc: felt252){
+            ${mapRange(cairoType.width, (n) => `WARP_STORAGE::write(${add('loc', n)}, 0);`).join(
+              '\n',
+            )}
```

Comment:
Let's do that in a variable and then just do `${var}`. It will be more readable

Reply:
I've just meant to do:
```let var = mapRange(...);
return [endent`
fn [../] {
 ${var}
`
```
a lot of brackets and backquotes within backquote string make it hard to read

Reply:
Oh ok, I thought you mean at cairo level. Changes incoming